apps/e2e/.env

```diff
@@ -23,7 +23,7 @@ APP_SECRET=
 # Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
 # IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml
 #
-DATABASE_URL="sqlite:///%kernel.project_dir%/var/data_%kernel.environment%.db"
+DATABASE_URL="sqlite:///%kernel.project_dir%/var/data.db"
```

Comment:
So we can generate the database in `APP_ENV=dev` (with Foundry enabled), and use it in `APP_ENV=prod`. 

This is not really something you would do for _real app_.